vercel/turbo
# Add log rotation to the daemon server

### Description

This PR expands our tracing setup to allow the daemon to opt-in to rotated log files on top of the regular stdout logging. This output is non-blocking and uses the standard tracing format, respecting the env vars set by the command that runs the daemon.

### Testing Instructions

Run `turbo daemon` with `TURBO_LOG_VERBOSITY` set to `trace`, and open the path it prints for the log file to see that the logs are also being output there.
